project/Build.scala

```diff
@@ -1440,6 +1459,73 @@ object Build {
       settings(commonBenchmarkSettings).
       enablePlugins(JmhPlugin)
 
+    def asScala3doc: Project = {
+      def generateDocumentation(targets: String, name: String, outDir: String, params: String = "") = Def.taskDyn {
+          val sourceMapping = "=https://github.com/lampepfl/dotty/tree/master#L"
+          run.in(Compile).toTask(s""" -o output/$outDir -t $targets -n "$name" -s $sourceMapping $params""")
+      }
+
+      project.settings(commonBootstrappedSettings).
+        dependsOn(`scala3-compiler-bootstrapped`).
+        dependsOn(`scala3-tasty-inspector`).
+        settings(
+          resolvers += Resolver.jcenterRepo,
+          resolvers += Resolver.bintrayRepo("kotlin", "kotlin-dev"),
+          resolvers += Resolver.bintrayRepo("virtuslab", "dokka"),
+          libraryDependencies ++= Seq(
+            "org.scala-lang" %% "scala3-tasty-inspector" % scalaVersion.value,
+
+            "com.virtuslab.dokka" % "dokka-site" % "0.1.9",
+            "com.vladsch.flexmark" % "flexmark-all" % "0.42.12",
+            "nl.big-o" % "liqp" % "0.6.7",
+            "args4j" % "args4j" % "2.33",
+
+            "org.jetbrains.dokka" % "dokka-test-api" % "1.4.10.2" % "test",
+            "com.novocode" % "junit-interface" % "0.11" % "test",
+          ),
+          test.in(Test) := test.in(Test).dependsOn(compile.in(Compile).in(`scala3doc-testcases`)).value,
```

Comment:
I think `test.in(Test)` is the same as `Test / test`, the latter is the recommended style now.